### PR TITLE
Revise 1 CDC FAQ for Symptoms & Testing

### LINKED ISSUE #570: Revise 1 CDC FAQ for Symptom & Testing
Request to revise FAQ: "What kind of test is being used to diagnose if I have COVID-19?"
Source file is [here](https://drive.google.com/open?id=1S8LJ54sJGbPO6quJXv6xL0yzIHXudGpW)

### PR DESCRIPTION
**Pull request by Mtcarp**
Fixes #570 
- Revise 1 CDC FAQ "What kind of test is being used to diagnose if I have COVID-19?" to  reflect changes sent by agency content lead
- Deletes change made/ merged from PR #564 (where content from FDA FAQ "What is the difference between the types of tests available for SARS-CoV-2?" picked up by site tracker was combined with this existing questions)

### CODE REVIEW

**Comment by Mtcarp** [comment]
@abbeykos This PR is ready for review. Once it's approved and merged, can you alert dev on duty to publish it to production so we can share the link with Misty. Thanks!

**Comment by brentryanjohnson** [comment]
It looks like the `title` was accidentally removed, @Mtcarp. [Preview](https://cg-dae9433e-23a0-46d7-bafb-dc79d2d3756c.app.cloud.gov/preview/18f/cv_faq/cdc-update-4-23/symptoms-and-testing/kind-of-test-used-to-diagnose-covid19/)

File: _content/symptoms-and-testing/kind-of-test-used-to-diagnose-covid19.md [hunk 1]
```diff
@@ -1,23 +1,16 @@
...
+date: April 22, 2020
 excerpt: Symptoms and testing
 layout: post
 promoted: false
 sources:
 - agency: cdc
   url: https://www.cdc.gov/coronavirus/2019-ncov/faq.html#Symptoms-&-Testing
-- agency: fda
-  url: https://www.fda.gov/emergency-preparedness-and-response/coronavirus-disease-2019-covid-19/coronavirus-disease-2019-covid-19-frequently-asked-questions
-title: What kind of test is being used to diagnose if I have COVID-19?
```
Comment (brentryanjohnson) [inline _content/symptoms-and-testing/kind-of-test-used-to-diagnose-covid19.md:12]: It looks like the title is missing.

Comment (Mtcarp) [reply _content/symptoms-and-testing/kind-of-test-used-to-diagnose-covid19.md:12]: Fixed; see preveiw: https://cg-dae9433e-23a0-46d7-bafb-dc79d2d3756c.app.cloud.gov/preview/18f/cv_faq/cdc-update-4-23/symptoms-and-testing/

**Comment by Mtcarp** [comment]
See [preview](https://cg-dae9433e-23a0-46d7-bafb-dc79d2d3756c.app.cloud.gov/preview/18f/cv_faq/cdc-update-4-23/symptoms-and-testing/) of this change.

**Pull request merged**